### ISSUE TITLE
Subscribe to local folder?

### ISSUE THREAD

**Issue by dar72** [einmalfel/PodListen#12]
Is it possible to add a feature to allow listing audio files in a folder on the device as if they were in a feed? I want to be able to listen to audio books with the same app that I listen to podcasts with.

This is something that I can do with BeyondPod version 3.3 which I want to replace because it's out of date now, I won't use their latest version because they did a horrible new "material design" interface.

**Comment by einmalfel** [comment]
Hello, @dar72 
I've never used such feature in podcast apps before, so I hardly can imagine how it could appear in PodListen. Could you please explain the feature with more details?
Do you want PodListen to add N files from folder to playlist on each refresh?

**Comment by dar72** [comment]
Yes, that's what it is, just like a normal audio player would do, the user adds to local folder into the list of subscriptions and each time new episodes are downloaded, the app also checks that local folder as well as the remote feeds.

It's just because I consider podcasts and audio books to be similar things and it's good to be able to listen to them using the same app. I don't listen to music on my phone so don't want to install a separate audio player just for occasional audio books. If I did listen to music on my phone, I wouldn't want my audio books mixed in with my music anyway.
